docs/en/migration.md

```diff
@@ -0,0 +1,20 @@
+Along with the release of OpenMMLab 2.0, MMDetection3D (namely MMDet3D) 1.x made many significant changes, resulting in less redundant, more efficient code and a more consistent overall design. These changes break backward compatibility. Therefore, we prepared this migration guide to make the transition as smooth as possible so that all users can enjoy the productivity benefits of the new MMDet3D and the entire OpenMMLab 2.0 ecosystem.
+
+## Environment
+
+MMDet3D 1.1 depends on the new foundational library for training deep learning models [MMEngine](https://github.com/open-mmlab/mmengine), and therefore has an entirely different dependency chain compared with MMDet3D 1.0. Even if you have a well-rounded MMDet3D 1.0 environment before, you still need to create a new python environment for MMDet3D 1.1. We provide a detailed [installation guide](./get_started.md) for reference.
```

Comment:
Det3D 1.1 depends on the new foundational library [MMEngine](https://github.com/open-mmlab/mmengine) for training deep learning models,

Reply:
compared with MMDet3D 1.0 and 0.x